pages/docs/couple-your-code/couple-your-code-gradient-data.md

```diff
@@ -75,7 +75,7 @@ double* stressGradient = new double[vertexSize * dim * dim]
 precice.initialize();
 
 while (not simulationDone()){ // time loop
-  precice.readBlockVectorData(displID, vertexSize, vertexIDs, displacements);
+  precice.readBlockVectorData(displID, vertexSize, vertexIDs, preciceDt, displacements);
```

Comment:
This code example has no `dt`, maybe you could remove the complete reading here.

Reply:
Completely removing the reading here is a different issue. Let's use `preciceDt = precice.getMaxTimeStepSize()` here, because this is also what we tell our users, if they don't want to worry about the `dt`

Reply:
I would actually try to make this example more consistent with the other examples. Before, the whole topic of picking the right time step size was completely missing. Let's keep it consistent and just do what we always do:

```
preciceDt = precice.getMaxTimeStepSize();
solverDt = beginTimeStep(); // e.g. compute adaptive dt
dt = min(preciceDt, solverDt);
```

We already introduce it this way much earlier (Step 2) and I think if we suddenly leave it out here, it does not really help.